_includes/silicon-valley/_interaction_area.html

```diff
@@ -5,6 +5,6 @@
 </div>
 
 <div class="secondary-buttons">
-  <button>Home</button>
-  <button>More info</button>
+  <button><i class="fas fa-home"></i> <span>Home</span></button>
```

Comment:
```suggestion
  <button><i class="fas fa-home"></i> <span>Start over</span></button>
```
Gary confirmed this is the language we'll be using since we're not designing a home page for these.